SightLocator/src/com/iolab/sightlocator/GetTextOnMarkerClickAction.java

```diff
@@ -37,6 +37,9 @@ public int describeContents() {
 		// TODO Auto-generated method stub
 		return 0;
 	}
+	
+	    public static final double MB = 2;
```

Comment:
Values are better to set in diferent file (strings.xml, for example).
In this case : 
<item type="integer" name="mega_byte">2</item>

and to retrieve in code:
getResources().getInteger(R.integer.mega_byte)